sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatConfig.java

```diff
@@ -0,0 +1,6 @@
+package com.azure.android.communication.chat;
+
+public class ChatConfig {
```

Comment:
Recommend making this package-private unless you really intend to expose these constants to the user.